recipe/meta.yaml

```diff
@@ -17,31 +17,40 @@ build:
 requirements:
   host:
     - pip
-    - setuptools
-    - wheel
     - python
+    - wheel
+    - setuptools >=61.0.0
   run:
     - python
     - numpy >=1.23.2
     - pandas >=2.0.0
     - attrs >=19.3.0
     - networkx >=2.4
     - multimethod >=1.4
+    - puremagic
 
 test:
+  source_files:
+    - pytest.ini
+    - tests
   imports:
     - visions
   commands:
     - pip check
+    - pytest -v -m "not spark_test"
   requires:
     - pip
+    - pytest >=5.2.0
+    - shapely
+    - pillow
+    - imagehash
 
 about:
   home: https://github.com/dylan-profiler/visions
+  summary: Type System for Data Analysis in Python
   license: BSD-4-Clause
```

Comment:
An interesting point of this license https://github.com/dylan-profiler/visions/blob/develop/LICENSE#L13.